packages/next-frontend/src/components/create-campaign/form/amm-step/index.tsx

```diff
@@ -22,21 +22,32 @@ interface AmmStepProps {
 
 export function AmmStep({ disabled, amm, onAmmChange }: AmmStepProps) {
     const t = useTranslations("new_campaign.form.amm");
+    const [open, setOpen] = useState(false);
     const availableAmms = useAvailableAmms();
 
     const getAmmChangeHandler = useCallback(
         (newAmm: AmmInfo) => {
             return () => {
                 if (amm && amm.slug === newAmm.slug) return;
                 onAmmChange({ amm: newAmm });
+                setOpen((open) => !open);
             };
         },
         [amm, onAmmChange],
     );
 
+    const handleStepOnClick = useCallback(() => {
```

Comment:
In this specific case I don't think we need a `useCallback` as there are no dependencies

Reply:
Fixed in 37d1fe105c5501a9d1d968990c6856b08fd8738d.